src/NzbDrone.Core/Datastore/Migration/158_enable_indexer_tags.cs

```diff
@@ -0,0 +1,16 @@
+using System.Data;
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(158)]
+    public class enable_indexer_tags : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("Indexers")
```

Comment:
Single line or align the periods.

Reply:
I can't see a VS setting for this and this is how it formatted automatically on save. Trying to align everything is going to be awkward so I'll just change this to a single line.

I'm curious why this style though. This is the only time I've seen C# code using this style and seems to be fighting against the editor. Typically indentation is always 4 spaces in. Keeping the periods aligned seems more similar to the typical conventions for scripting languages.

If I'm just missing a VS setting to auto align the periods, let me know and I'll add it to the editorconfig.

Reply:
It might just be a resharper option. We have a bit of a mix but a single line works well here (or make it four lines for each part of the fluent chain) (I see it's one line now 👍)